packages/table/src/Controls/TableControls.tsx

```diff
@@ -1,20 +1,33 @@
 import React, { cloneElement, isValidElement, useEffect, useState } from 'react';
 import { Form } from 'reactstrap';
+import { useTableContext } from '../TableContext';
 
 type Props = {
   id?: string;
   disabled?: boolean;
   children?: React.ReactNode[] | React.ReactNode;
 } & React.HTMLAttributes<HTMLElement>;
 
-const TableControls = ({ id, disabled, children, ...rest }: Props): JSX.Element => {
+const TableControls = ({ id, disabled, children, ...rest }: Props): JSX.Element | null => {
   const [isDisabled, setIsDisabled] = useState(disabled || false);
 
+  const { instance } = useTableContext();
+  const [isReady, setIsReady] = useState(!!instance);
```

Comment:
I don't think you need to use state for this. Instead, you can change your check to see if `instance` exists.